Timeline.cpp

```diff
@@ -62,6 +82,7 @@ bool TweenDuino::Timeline::add(TweenDuino::Tween &tween) {
 
     TweenDuino::Timeline::TimelineEntry &entry = tweens[entryIndex];
     if (entry.tween != nullptr) {
+		delete &tween;
```

Comment:
I think this line should be removed.

I can see why you'd want to delete this here; it's annoying to have to check the return from Timeline::add() and potentially de-allocate the tween if add() returns false.  It's more concise to do it here than in Timeline::addTo().

However, deleting in Timeline::add() means that someone using the library doesn't have any say in what happens to the memory they allocated themselves if they call Timeline::add() and I suspect could cause some unexpected crashes because it's kinda unexpected behavior for an "add" function.

For now I lean towards adding checks and deletions to the Timeline::addTo() functions.  Something like
```
    if (!add(*tween)) {
        // Oops, no room!
        delete tween;
        tween = nullptr;
    }
```

Reply:
I leaked memory if additional tweens are attempted to be added past the max in my tests. This was actually the last leak I found and only by chance due to accidently having a double Timeline::addTo line.